packages/headless-inspector/package.json

```diff
@@ -35,6 +35,7 @@
   "devDependencies": {
     "@types/express": "4.17.13",
     "@types/ws": "8.5.3",
+    "@types/yargs": "16.0.8",
```

Comment:
This project depended on `@types/yargs`, but it was not explicitly installed in `package.json`.  
As a result, when `jest-environment-jsdom` is installed, the version of `@types/yargs` changes.

To solve this, I modified it so that `@types/yargs` is explicitly installed.